lib/helpers.js

```diff
@@ -42,6 +56,16 @@ function logError(logString) {
   } else {
     // Prepend the timestamp in production
     console.error(logString) // eslint-disable-line no-console
+    Sentry.captureException(logString)
+  }
+}
+
+function logSentry(logString) {
+  if (isTestEnvironment()) {
+    // Output in colour in test
+    console.log(chalk.dim.cyan(`\t${logString}`)) // eslint-disable-line no-console
+  } else {
+    Sentry.captureException(logString)
```

Comment:
@mariocimet @trisapeace does Sentry have a way to distinguish between log messages and error messages? is `captureException` the only way to submit logs to Sentry?

Reply:
Looks like capturing messages is also a thing: https://docs.sentry.io/platforms/node/usage/#capturing-messages